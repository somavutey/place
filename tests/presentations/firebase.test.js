import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import useFirebaseAuth from "./../../components/presentations/firebase";

it("firebase", () => {
  const wrapper = shallow(<useFirebaseAuth />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
