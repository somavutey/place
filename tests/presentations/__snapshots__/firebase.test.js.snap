// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`firebase 1`] = `<useFirebaseAuth />`;
